docs/DeveloperGuide.md

```diff
@@ -862,15 +860,15 @@ testers are expected to do more *exploratory* testing.
 
 ### Deleting a person
```

Comment:
Since you are changing all the `person` to `contact`, perhaps this should be changed as well.

Reply:
Ok, missed that out